app/src/main/java/org/eyeseetea/malariacare/EyeSeeTeaApplication.java

```diff
@@ -45,6 +46,7 @@ public void onCreate() {
         Fabric.with(this, new Crashlytics());
         LocationMemory.getInstance().init(getApplicationContext());
         // Not previously populated tables
+        Session.setUIDPROGRAM(getResources().getString(R.string.UID_PROGRAM));
```

Comment:
@idelcano Why are you storing this uid in the session? Maybe it is better to read it [like this](https://github.com/EyeSeeTea/malariapp/blob/master/app/src/main/java/org/eyeseetea/malariacare/network/PushClient.java#L175) as we discussed.